### PR TITLE
grub/help/en_US: Fix typos

### PR DESCRIPTION
**Pull request by kbdharun**

### CODE REVIEW

File: INSTALL/grub/help/en_US.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
  L - Language Select
 F1 - Display this help information
```
Comment (kbdharun) [inline INSTALL/grub/help/en_US.txt:2]: I am not sure about this one, I will take a look at it in the latest release and update here

```suggestion
F1 - Display help information
```

Comment (kbdharun) [reply INSTALL/grub/help/en_US.txt:2]: Edit: Seems like the release was put on hold. So if this change makes sense feel free to commit it.

File: INSTALL/grub/help/en_US.txt [hunk 2]
```diff
@@ -11,8 +11,8 @@ m/Ctrl+m - Checksum image files (md5/sha1/sha256/sha512)
 d/Ctrl+d - Memdisk Mode (Only for small WinPE/LiveCD ISO/IMG)
 w/Ctrl+w - WIMBOOT Mode (Only for standard Windows ISO)
 r/Ctrl+r - Grub2 Mode (Only for some Linux distros)
-i/Ctrl+i - Compatible Mode (Just only for debug)
-u/Ctrl+u - Load ISO efi driver (Just only for debug, can not be used officially)
+i/Ctrl+i - Compatible Mode (Just only for debugging)
```
Comment (wiz64) [inline INSTALL/grub/help/en_US.txt:14]: Removing Just would make it shorter and simpler
```suggestion
i/Ctrl+i - Compatible Mode (Only for debugging)
```
```suggestion
u/Ctrl+u - Load ISO EFI driver (Only for debugging, can not be used officially)
```

Comment (kbdharun) [inline INSTALL/grub/help/en_US.txt:14]:
```suggestion
i/Ctrl+i - Compatible Mode (Only for debugging)
```

File: INSTALL/grub/help/en_US.txt [hunk 3]
```diff
@@ -11,8 +11,8 @@ m/Ctrl+m - Checksum image files (md5/sha1/sha256/sha512)
...
+u/Ctrl+u - Load ISO EFI driver (Just only for debugging, can not be used officially)
```
Comment (kbdharun) [inline INSTALL/grub/help/en_US.txt:15]:
```suggestion
u/Ctrl+u - Load ISO EFI driver (Only for debugging, can not be used officially)
```

**Review by kbdharun — COMMENTED** [review]
I couldn't see your comment, but this seems fine too I guess @wiz64 .

**Comment by wiz64** [comment]
Just remove the "Just" and

**Comment by kbdharun** [comment]
> Just remove the "Just"s and
> 
> ```
> F1 - Display help information
> ```
> 
> seems fine

Done

**Comment by steve6375** [comment]
```
F1 - Display this help information
F2 - Browse and boot a file from a local disk
F3 - Switch menu mode between Treeview <-> ListView
F4 - Search and boot Windows/Linux/xorboot from local disks
F5 - Utilities
F6 - Load custom Grub2 Menu (/ventoy/ventoy_grub.cfg)
F7 - Switch between GUI Mode <-> TEXT Mode
l/Ctrl+l - Select Language
m/Ctrl+m - Checksum selected file (md5/sha1/sha256/sha512)
d/Ctrl+d - Memdisk Mode (use for small WinPE/LiveCD ISO/IMG)
w/Ctrl+w - WIMBOOT Mode (use for standard Windows ISO)
r/Ctrl+r - Grub2 Mode          (use for some Linux distros)
i/Ctrl+i - Compatible Mode     (debug use only)
u/Ctrl+u - Load ISO efi driver (debug use only)
```

Shouldn't we also have Ctrl+l for language ???

is Ctrl+u correct description ???  It just shows 'UEFI FS' on menu???

**Comment by wiz64** [comment]
> ```
> F1 - Display this help information
> F2 - Browse and boot a file from a local disk
> F3 - Switch menu mode between Treeview <-> ListView
> F4 - Search and boot Windows/Linux/xorboot from local disks
> F5 - Utilities
> F6 - Load custom Grub2 Menu (/ventoy/ventoy_grub.cfg)
> F7 - Switch between GUI Mode <-> TEXT Mode
> l/Ctrl+l - Select Language
> m/Ctrl+m - Checksum selected file (md5/sha1/sha256/sha512)
> d/Ctrl+d - Memdisk Mode (use for small WinPE/LiveCD ISO/IMG)
> w/Ctrl+w - WIMBOOT Mode (use for standard Windows ISO)
> r/Ctrl+r - Grub2 Mode          (use for some Linux distros)
> i/Ctrl+i - Compatible Mode     (debug use only)
> u/Ctrl+u - Load ISO efi driver (debug use only)
> ```
> 
> Shouldn't we also have Ctrl+l for language ???
> 
> is Ctrl+u correct description ??? It just shows 'UEFI FS' on menu???

We're discussing Grammer here, not keybindings.. longpanda manages the key binding stuff

**Comment by steve6375** [comment]
ctrl+l is already supported and working. It is just not mentioned in Help menu

**Comment by wiz64** [comment]
@ventoy add ctrl + l as well ?

> ctrl+l is already supported and working. It is just not mentioned in Help menu

**Comment by kbdharun** [comment]
> ctrl+l is already supported and working. It is just not mentioned in Help menu

Oh, Then I will add it too if necessary.

**Comment by wiz64** [comment]
@kbdharun line 15, change "can not" to "cannot" ?

**Pull request merged**